specification/paths/ExportRatesTemplate.json

```diff
@@ -0,0 +1,67 @@
+{
+  "post": {
+    "tags": [
+      "RPC"
+    ],
+    "security": [
+      {
+        "OAuth2": [
+          "organizations.manage"
+        ]
+      }
+    ],
+    "summary": "Export service rates template.",
+    "description": "This endpoint retrieves a template based on existing service rates, this can then be used to edit and add existing and new service rates into the system.",
```

Comment:
```suggestion
    "description": "This endpoint retrieves a template based on existing service rates, this can then be used to edit existing and add new service rates into the system.",
```